src/components/product.jsx

```diff
@@ -15,26 +15,23 @@ const Product = (props) => {
       
       <div className="counterButtonWrapper">
         <button
-          onClick={() => {
-            count--
-            setCount(count)
-          }}
+          onClick={props.onDecrement}
+          // QUESTION: what is wrong with my syntax here (and line 28)? my goal is to grab the current count (props.count) for an individual product, pass it up to ProductTable, and finally up to ProductPage
```

Comment:
> for an individual product

Ok I think I know what the issue is but I'm not exactly sure what you are trying to accomplish. My assumption is though, that you have a single [count, setCount] state but you want it to manage multiple products. If you want each product to have it's own count, each would need it's own piece of state... I would store the count of each product in the list itself. So basically when you get the product list, save it to state with its count (initialized to 0) and update the product list with the count. Alternatively, you could keep a separate list of products that have a count that is more than zero in state. Either way, you need to link a count to an individual product.